### ISSUE TITLE
Customize hotkeys

### ISSUE THREAD

**Issue by eddy-geek** [franziskuskiefer/app-keys-gnome-shell-extension#25]
This is a feature request for customization within the extension (often requested on extensions.gnome.org).

Workaround: I tried to customized the keybindings myself by declaring gnome bindings that "forward" to the standard binding through xdotool ([see this gist](https://gist.github.com/eddy-geek/c8ecb982974db7e0b9d7)) and it partially works: when using the "new" binding a new instance is always launched, i.e. we lose the "activate existing" and "cycle through" features. The only program I found immune to this bug is gnome's Files.

**Comment by franziskuskiefer** [comment]
I agree that this would be a nice feature. But I don't know when I'll have time to get to it (PRs are as always welcome).

**Comment by eddy-geek** [comment]
Hopefully I'll find the time...

In the meantime, do you have any idea why both "xdotool key super+6" and "xdotool key super+KP_6" (from the terminal or Alt+F2) would act differently (always create new window) from pressing the actual keys on the keyboard ?

**Comment by franglais125** [comment]
Hi, I don't know if there is interest in this, but I implemented something very similar for dash-to-panel.

Here is the commit: https://github.com/jderose9/dash-to-panel/commit/015e9c247f779f83472e509f1f513c1a98b8ab56. The part of interest is the function "_checkHotkeyPrefix".

**Comment by eddy-geek** [comment]
@franglais125 yes, this will be very helpful to anyone willing to PR this.

I'm still using my xdotool workaround, but as there's no xdotool on wayland a proper solution will be needed soon ;-)
